### PR TITLE
Setup the dev environment

### LINKED ISSUE #1: Inital repo and dev dependencies
### User Story
As a developer I need a good developer environment to create an app. 

### Clarification
At the moment i start with my Gesellenstück and need a developer environment to start with programming. 

### To do
- [ ] Repo at GitHub
- [ ] Clone Repo via Terminal to Visual Studio Code
- [ ] Install React, Prettier, ESlint, Express, Emotion, Husky, Storybook, Now, Nodemon, Deepscan
- [ ] Create Pull Request

### PR DESCRIPTION
**Pull request by meloneminze**
Set up the project, a server split and add the deployment with heroku. 
Closes #1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,68 @@
-# famly
-Gesellenstück Neue Fische 
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
```
Comment (lmachens) [inline README.md:1]: This Readme and the available scripts noticed belongs to the client package.json. 
Maybe move this file and create a new README.md for the top-level scripts.

File: .gitignore [hunk 2]
```diff
@@ -102,3 +102,26 @@ dist
 
 # TernJS port file
 .tern-port
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# dependencies
+/node_modules
```
Comment (lmachens) [inline .gitignore:108]: The `client/node_modules` folder is not ignored

File: README.md (hunk 1 above)
Comment (meloneminze) [reply README.md:1]: Thank you for your review. I moved this file and created a new one for the top-level scripts.

File: .gitignore (hunk 2 above)
Comment (meloneminze) [reply .gitignore:108]: It is, because i also ignore `node_modules/`in the gitignore. I will delete the ´/node_modules`.

**Pull request merged**